mmseg/utils/collect_env.py

```diff
@@ -41,9 +42,12 @@ def collect_env():
         for name, devids in devices.items():
             env_info['GPU ' + ','.join(devids)] = name
 
-    gcc = subprocess.check_output('gcc --version | head -n1', shell=True)
-    gcc = gcc.decode('utf-8').strip()
-    env_info['GCC'] = gcc
+    try:
+        gcc = subprocess.check_output('gcc --version | head -n1', shell=True)
+        gcc = gcc.decode('utf-8').strip()
+        env_info['GCC'] = gcc
+    except subprocess.CalledProcessError:
+        env_info['GCC'] = 'n/a'
```

Comment:
Are these lines necessary?

Reply:
Necessary under Windows, but not under Linux.

Reply:
These could be in another PR. 
We may omit these in this PR.

Reply:
OK.